src/styles/styles.js

```diff
@@ -3931,6 +3931,18 @@ const styles = {
         overflow: 'hidden',
     },
 
+    mapViewOverlay: {
+        flex: 1,
+        position: 'absolute',
+        left: 0,
+        top: 0,
+        borderRadius: variables.componentBorderRadiusLarge,
+        overflow: 'hidden',
+        backgroundColor: themeColors.highlightBG,
```

Comment:
Couples of actions failed here. @akinwale Could check? Thanks!

Reply:
@mollfpr Looks like main got updated with theme switching, see https://expensify.slack.com/archives/C01GTK53T8Q/p1694661377285649 for reference.

I have merged with latest main and fixed the issue.